Services/Container/classes/class.ilContainerGUI.php

```diff
@@ -3595,10 +3595,13 @@ public function initFormTitleDescription(ilPropertyFormGUI $form)
 		$desc = new ilTextAreaInputGUI($this->lng->txt("description"), "desc");
 		$desc->setRows(2);
 		$desc->setCols(40);
-		if(is_object($trans))
-		{
-			$desc->setValue($trans->getDefaultDescription());
+		$descriptionValue = $this->object->getLongDescription();
+		if (is_object($trans)) {
+			if (strlen($descriptionValue) < strlen($trans->getDefaultDescription())) {
```

Comment:
Maybe we could also check if the initial value of shorter value is a sub string of the longer value. If yes, use the long description. If not, use the (maybe shorter) description from `object_translation`.